cmd/main.go

```diff
@@ -60,7 +60,7 @@ var (
 )
 
 const (
-	SiteConfigNamespace               = "siteconfig-system"
+	SiteConfigNamespace               = "siteconfig-operator"
```

Comment:
You should be pulling this from the environment.

Not sure what you're using it for, but this is going to break when this is deployed by MCE in `multicluster-engine`.

Reply:
Good point! I use it to create the default ConfigMap templates for AI and IBI. Perhaps we should create a separate namespace `siteconfig-templates` and create these ConfigMaps in there? I don't think it would be good to create them in the mce namespace. wdyt?

Reply:
For now I'd say put them in the namespace the operator is running in.

I'm not sure about creating a new namespace. Maybe ask for some opinions on this in slack or in our sync meeting.

Reply:
Done using the downward API.